### DISCUSSION TITLE
Raspberry pi 3b+ x64

### DISCUSSION THREAD

**Discussion by keukenrol** [rtl-airband/RTLSDR-Airband#351] [Q&A]
Hi

I have the x64 version of raspbian installed on my 3b+. Since its a armv8 WITH a videocore iv, which compilation option would be best?

Thanks!

**Comment by charlie-foxtrot** [comment]
The default may be just fine or try `native`.  If you want to try different profiling options you can use `rtl_sdr` to create a raw capture from your SDR then use that same file with different test builds.
